src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/CvlRecordService.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.dates.ReleaseDateService
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonerSearchPrisoner
+
+@Service
+class CvlRecordService(
+  private val eligibilityService: EligibilityService,
+  private val releaseDateService: ReleaseDateService,
+) {
+
+  fun getCvlRecord(prisoner: PrisonerSearchPrisoner, areaCode: String): CvlCaseDto {
+    val recordList = getCvlRecords(listOf(prisoner), mapOf(prisoner.prisonerNumber to areaCode))
```

Comment:
nitpick: getCvlRecord but we return a CvlCaseDto, shoul'nt these share a common name,...